pkg/controller/perconaservermongodb/smart.go

```diff
@@ -3,6 +3,7 @@ package perconaservermongodb
 import (
 	"context"
 	"fmt"
+	"github.com/percona/percona-server-mongodb-operator/pkg/psmdb/backup"
```

Comment:
goimport file pls)

Reply:
updated in 2ab5eebbccd167792aea4b69b5fae6680f19e217